libclamav/others.c

```diff
@@ -421,6 +421,11 @@ cl_error_t cl_init(unsigned int initoptions)
 
     UNUSEDPARAM(initoptions);
 
+    /* Rust logging initialization */
+    if (!clrs_log_init()) {
+        cli_dbgmsg("rust logging initialization failed");
```

Comment:
I might not know enough about ClamAV logging today, but should we maybe have a little more stern of an indication that something went wrong here?

Reply:
Yeah we could probably bump this to `cli_warnmsg` or even `cli_errmsg`

Oh, this is also missing a `\n` new line.